scripts/_docker.sh

```diff
@@ -76,6 +78,65 @@ function _docker_build_with_custom_tag() {
     cd $root
 }
 
+function _docker_update_service() {
+    local account=$1
+    local env=$2
+    local service=$3
+
+    if [[ -z ${account} ]]; then
+        echo "Account is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${env} ]]; then
+        echo "Env is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${service} ]]; then
+        echo "Service (back-end|ingestion|front-end) is required" >&2
+        return 1
+    fi
+
+    uhd docker ecr:login tools
+
+    local latest_image_tag
+    case "${service}" in
+        "back-end")
+            latest_image_tag=$(_docker_get_most_recent_back_end_image_tag)
+            ;;
+        "ingestion")
+            latest_image_tag=$(_docker_get_most_recent_ingestion_image_tag)
+            ;;
+        "front-end")
+            latest_image_tag=$(_docker_get_most_recent_front_end_image_tag)
+            ;;
+    esac
```

Comment:
could do with a little error handling:-
```suggestion
    case "${service}" in
        "back-end") latest_image_tag=$(_docker_get_most_recent_back_end_image_tag) ;;
        "ingestion") latest_image_tag=$(_docker_get_most_recent_ingestion_image_tag) ;;
        "front-end") latest_image_tag=$(_docker_get_most_recent_front_end_image_tag) ;;
        *) echo "Invalid service name '${service}'" >&2; return 1 ;;
    esac
```